### PR TITLE
Animals

### PR DESCRIPTION
**Pull request by patriciachrysy**
Created database, tables and requests

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
+CREATE DATABASE vet_clinic;
 
 CREATE TABLE animals (
-    name varchar(100)
+    id int,
```
Comment (tufoinnkuo10) [inline schema.sql:5]: [OPTIONAL] you can  mark the id column as primary key
`PRIMARY KEY (id)`

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,20 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+
+
+SELECT name FROM animals WHERE neutered = true AND  escape_attempts >= 3;
```
Comment (tufoinnkuo10) [inline queries.sql]: - [x] it can list the name of all animals that are neutered and have less than 3 escape attempts, not greater than 3. Kindly fix that

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @patriciachrysy 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ Documented your work professionally
✅ Great use of screenshots

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by prolajumokeoni — APPROVED** [review]
Hi @patriciachrysy 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-  Adding  screenshots of queries results to the pull request description.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**